_ml-commons-plugin/api/mcp-server-apis/delete-mcp-tools.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Delete MCP Tools 
+parent: Agent APIs
+grand_parent: ML Commons APIs
+nav_order: 50
+---
+
+# Delete MCP Tools
+**Introduced 3.0**
+{: .label .label-purple }
+
+You can use this API to delete one or more MCP tools with tool name.
+
+## Endpoints
+
+```json
+POST /_plugins/_ml/mcp/tools/_remove
```

Comment:
I don't think this is a correctly formatted request. Should this be:
```
POST /_plugins/_ml/mcp/tools/_remove
{
  "tools": [
    "WebSearchTool", 
    "ListIndexTool"
  ]
}
```
?

Reply:
It is a formatted request, array list is also a valid json and this is working in our code.